### ISSUE TITLE
Output css file modifies the style of the whole page

### ISSUE THREAD

**Issue by NelDav** [tfachmann/MdCollab#28]
The css file for the output html div also changes the style of the other site elements.

**Comment by NelDav** [comment]
I think the output html should be placed into a `iframe`. It should be possible to apply a different css file for the document presented by the `iframe`.

**Comment by tfachmann** [comment]
Yeah the `css` file for the output should be bound to a scope just containing the HTML output

**Comment by NelDav** [comment]
The `iframe` element includes a html file into a container. It is possible to include the css file into this html document. The style is only applied to the html of this document if you do this. 


But I did not find a solution to modify the html of an element in another file in javascript. I think the easiest way is to write this separate html file directly from rust.

**Comment by tfachmann** [comment]
This does not sound too smart to be honest.
I have found similar use-cases [here](https://stackoverflow.com/questions/17667874/limit-scope-of-external-css-to-only-a-specific-element) to limit the scope of the CSS file. However, the trivial way to do it has been dropped / is deprecated.

**Comment by NelDav** [comment]
Regarding to this post on [GitHub](https://github.com/rustwasm/wasm-bindgen/issues/1727#issuecomment-522697904_) and this one on [Stackoverlow](https://stackoverflow.com/a/53305717/6911200) the std::fs library is not suppoerted by wasm so far.

[fetch](https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.Window.html#method.fetch_with_str) seems to be a alternative.

**Comment by NelDav** [comment]
There are several alternatives to read the content of a file. But I cannot find any alternative to write into a file. Maybe somebody could help searching for alternatives. It is not necessary to use the `iframe` method. If anybody find a solution to realize scoped css in another way that would be fine as well.

**Comment by NelDav** [comment]
I found the _fs_ module to read and write files in server side javascript code. But I cannot use it. I always get the following error:

> Module not found: Error: Can't resolve 'fs' in '<site directory>'
> @ ./index.js 1:9-22

Could you test if the _fs_ module works on your system? You can find the documentation [here](https://nodejs.org/api/fs.html).
